src/view_tab.cpp

```diff
@@ -200,15 +201,21 @@ void View::DrawTabDurSym(DeviceContext *dc, LayerElement *element, Layer *layer,
 
         int dotSize = 0;
 
-        if (staff->IsTabGuitar()) {
+        if (tabGrp->IsInBeam() || staff->IsTabGuitar()) {
```

Comment:
Don't you want to have the same size for dots within and without beams in lute tablature? It seems that those within will now have guitar tab size (and position.)

Reply:
I'll look out some beamed tablatures with dots, but you are right we will probably need to adjust those as well.

Reply:
That is fine. Do you want me to merge this already, or wait? I am fine either way.

(PS I have to fix the additional stem showing up together with the glyph for lute tablature. I introduced the bug when working on the guitar tab. I can do that right after the merge)

Reply:
I can't find a single example in my tablature facsimiles of a dotted note under a beam!  But I've only looked at a small sample.  Experimenting using the same size and distancing for dots under a beam that I've implemented for flags looks too big and too far from the stem.   In any case as the beams don't currently look like renaissance tablature "gridiron" beams I think we can leave dots under beams as they are for later consideration.   I think it's ok to merge.

![test mei](https://user-images.githubusercontent.com/76966668/154040075-1a81a703-a522-4715-a118-828ee39e472a.svg)